src/gui/mainwindow.cpp

```diff
@@ -1489,6 +1489,21 @@ void MainWindow::showStatusBar(bool show)
     }
 }
 
+bool MainWindow::isFiltersSidebarVisible() const
+{
+    const QList<int> sizes = m_splitter->sizes();
+    return sizes[0] != 0;
+}
+
+void MainWindow::showFiltersSidebar(const bool show)
```

Comment:
The splitter handle should be disabled or hidden.

Reply:
The handle is now disabled when the sidebar is disabled but you can still resize the sidebar to 0 width.
This needs some investigation because we can't make the left panel non collapsible.